### DISCUSSION TITLE
Virtual Box from Kick start disk

### DISCUSSION THREAD

**Discussion by SadiqJohn** [utmapp/UTM#6585] [Ideas]
I'm not sure if its currently possible, but its not immediate apparent if creating a virtual box from a kick start disk is possible

**Comment by naveenrajm7** [comment]
Hello,

Virtual Box? do you mean Virtual Machine?
kick start disk? do you mean kickstart file for automated installation?

if you are asking for automated/unattended OS installation/VM creation. Please refer to https://github.com/utmapp/UTM/issues/5211.
In short there are ways to create VMs from kickstart file (or similar methods like preseed, autoinstall etc) in QEMU , which can be used in UTM aswell. For instance, using VNC. 

You can manually configure the boot commands for your VM in UTM to use kickstart file. 
In future, this can be automated as UTM exposes more APIs, keep an eye on the above-mentioned issue and check out the packer plugin for UTM to build UTM VMs which will eventually add support for kickstart installation once required UTM API is exposed.
https://github.com/naveenrajm7/packer-plugin-utm
